BSB_LAN/BSB_LAN_EEPROMconfig.h

```diff
@@ -206,9 +206,9 @@ const configuration_struct config[]={
   {CF_PASSKEY,          2, CCAT_IPV4,     CPI_TEXT,      CDT_STRING,         OPT_FL_ADVANCED, CF_PASSKEY_TXT, sizeof(PASSKEY)},//immediately apply
   {CF_TRUSTEDIPADDRESS, 2, CCAT_IPV4,     CPI_TEXT,      CDT_IPV4,           OPT_FL_ADVANCED, CF_TRUSTEDIPADDRESS_TXT, sizeof(trusted_ip_addr)}, //immediately apply
   {CF_TRUSTEDIPADDRESS2,2, CCAT_IPV4,     CPI_TEXT,      CDT_IPV4,           OPT_FL_ADVANCED, CF_TRUSTEDIPADDRESS_TXT, sizeof(trusted_ip_addr2)},//immediately apply
-#ifdef ESP32
+  #ifdef ESP32
```

Comment:
Indention was correct before. It does not follow code-level indention, but compiler-directive-level indention, and there is no #if clause before

Reply:
Sorry, when moving the config setting back to its original place, I copied the indentation for the preprocessor directives from line 179 ("When in Rome, do as the Romans."). I'll fix both locations now.